srv/components/controller/files/python-scripts/xml.py

```diff
@@ -0,0 +1,91 @@
+import logging
```

Comment:
`srv/components/controller/files/python-scripts/xml.py`
`python-scripts` is not necessary, just call it `scripts`

Also, rename the file to: `srv/components/controller/files/scripts/xml_parser.py`

Do we really need this file? I feel we don't need to have a separate parser module in Python, we can use standard xml parser.

Secondly, we don't need xml as we can capture output as JSON. @83bhp need your help with JSON